src/test/storage/encoded_column_test.cpp

```diff
@@ -17,32 +21,17 @@
 
 namespace opossum {
 
-namespace hana = boost::hana;
-
-using EncodingTypes = ::testing::Types<enum_constant<EncodingType, EncodingType::Dictionary>,
-                                       enum_constant<EncodingType, EncodingType::DeprecatedDictionary>,
-                                       enum_constant<EncodingType, EncodingType::RunLength>>;
-
-template <typename EncodingTypeT>
-class EncodedColumnTest : public BaseTest {
- protected:
-  static constexpr auto encoding_type_t = EncodingTypeT{};
-  static constexpr auto column_template_t = hana::at_key(encoded_column_for_type, encoding_type_t);
-
-  using ColumnTemplateType = typename decltype(column_template_t)::type;
-
-  template <typename T>
-  using EncodedColumnType = typename ColumnTemplateType::template _template<T>;
-
+class EncodedColumnTest : public BaseTestWithParam<ColumnEncodingSpec> {
```

Comment:
I changed this test from a templated to a parameterised one. Previously, `EncodingType::Dictionary` was only tested with the default `VectorCompressionType`, which is `FixedSizeByteAligned`. Now it also tests `SimdBp128`, which was buggy.